src/backend/javascript/ib_to_js.rs

```diff
@@ -11,7 +11,7 @@ pub fn build_block(block: &ib::Block) -> Result<Block> {
     for operation in &block.0 {
         match *operation {
             ib::Operation::ThrowError =>
-                b.expr(format!("throw \"error\"").into()),
+                b.expr(format!("throw new Error(\"Protodef error\")").into()),
```

Comment:
Error output will be: `Error: protodef error`, rather than `Error: Protodef error`